### ISSUE TITLE
some bollards and other items return different items/show different items when broken or using WAILA

### ISSUE THREAD

**Issue by InuYasha86000** [Flenix/FlenixRoads#60]
for the purpose of simplifying this bug I'm focusing on the yellow metal bollard, for some reason you can place it as it is, but once placed it's metadata I guess shows it as a concrete bollard and if you break it in survival or middle click it, you will get the concrete bollard instead.
there are other items in the same category that have this problem too however I have focused on this one as it's the easiest to see the issue without sifting through the others for now.
![2023-09-09_22 23 26](https://github.com/Fureniku/Furenikus-Roads/assets/15805183/2b09e5ce-7d56-4ac5-89cf-b677918551bd)

**Comment by InuYasha86000** [comment]
white speed bumps come out as black speed bumps as well.

**Comment by InuYasha86000** [comment]
so I just finally took apart roads mod a bit and I MIGHT have an idea what's happening with the changing of block ID's when they are placed (the yellow bollard saying it's a concrete one and breaking into a concrete one). 

I am NOT a mod creator so I may be wrong on this, but I noticed this Default section for bollards:
"defaults": {"transform": "forge:default-block","model": "furenikusroads:bollard_concrete_1"*

varriants include: smooth metal, yellow, ringed, black, as well as concrete, they all fall under that default "furenikusroads:bollard_concrete_1"
[11:39 AM]
my hunch is for some reason it's placing the texture data on the default data when placed, but then when the item is placed it's overwriting the actual item data with the default data in the section. i'm not sure but it's a hunch.
[11:43 AM]
i'm hoping that helps with the item changing when placed/broken issue.
